src/components/modules/setting/Settings.tsx

```
@@ -0,0 +1,25 @@
+"use client";
+
+import useSettings from "./hooks/settings.hook";
+import ProfileSection from "./profileSection";
+
+const Settings = () => {
+  const { currentTab, setCurrentTab, saveProfile, theme, toggleTheme } =
```

ESLint

[error] 'setCurrentTab' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)